lib/screens/authenticate/sign_up.dart

```diff
@@ -27,94 +29,79 @@ class _SignUpState extends State<SignUp> {
-                        setState(() => name = val);
-                      },
-                    ),
-                    SizedBox(height: 20.0),
-                    TextFormField(
-                      decoration:
-                          textInputDecoration.copyWith(hintText: 'email'),
-                      validator: (val) =>
-                          val.endsWith(innoMail) || val.endsWith(innoMail2)
-                              ? null
-                              : 'Enter an Innopolis email',
-                      onChanged: (val) {
-                        setState(() => email = val);
-                      },
-                    ),
-                    SizedBox(height: 20.0),
-                    TextFormField(
-                      decoration:
-                          textInputDecoration.copyWith(hintText: 'password'),
-                      obscureText: true,
-                      validator: (val) => val.length < 6
-                          ? 'Enter a password 6+ chars long'
-                          : null,
-                      onChanged: (val) {
-                        setState(() => password = val);
-                      },
-                    ),
-                    SizedBox(height: 20.0),
-                    RaisedButton(
-                        color: Colors.pink[400],
-                        child: Text(
-                          'Sign Up',
-                          style: TextStyle(color: Colors.white),
-                        ),
-                        onPressed: () async {
-                          if (_formKey.currentState.validate()) {
-                            setState(() => loading = true);
-                            User user = await _auth.signUp(
-                                email, password, name, yearId);
-                            if (user == null) {
-                              setState(() {
-                                loading = false;
-                                error = 'The email address is already in use';
-                              });
-                            } else {
-                              user.sendEmailVerification();
-                            }
-                          }
-                        }),
-                    SizedBox(height: 12.0),
-                    Text(
-                      error,
-                      style: TextStyle(color: Colors.red, fontSize: 14.0),
-                    )
-                  ],
-                ),
+        : new AuthLayout(
+            form: Form(
+              key: _formKey,
+              child: Column(
+                children: <Widget>[
+                  error != ''
+                      ? Text(
+                          error,
+                          style: TextStyle(
+                            color: Colors.red,
+                            fontSize: 14.0,
+                          ),
+                        )
+                      : Container(),
```

Comment:
the same comment as to the sign_in page

Reply:
ok, gonna fix

Reply:
Fixed
node: SizedBox.shrink() didn't work as expected